drivers/qwiic/micro_oled.c

```diff
@@ -98,13 +101,18 @@ static uint8_t micro_oled_screen_buffer[LCDWIDTH * LCDWIDTH / 8] = {0x00, 0x00,
 // TODO: generate bitmap of QMK logo here
 #    endif
 #else
-// catchall for custom screen szies
-static uint8_t micro_oled_screen_buffer[LCDWIDTH * LCDWIDTH / 8] = {0};
+// catchall for custom screen sizes
+static uint8_t micro_oled_screen_buffer[LCDWIDTH * LCDHEIGHT / 8] = {0};
 #endif
 
 void micro_oled_init(void) {
     i2c_init();
-    i2c_start(I2C_ADDRESS_SA0_1);
+
+    #ifdef __AVR__
+        i2c_start(I2C_ADDRESS_SA0_1, 100);
+    #else
+        i2c_start(I2C_ADDRESS_SA0_1);
+    #endif
```

Comment:
I'm not a fan of this difference int API.  

If anything, the i2c API should be unified, though that should be a separate PR, I think.

Reply:
+1 for separate PR